layer/layer.go

```diff
@@ -173,7 +173,7 @@ type Store interface {
 	Map() map[ChainID]Layer
 	Release(Layer) ([]Metadata, error)
 
-	CreateRWLayer(id string, parent ChainID, mountLabel string, initFunc MountInit, storageOpt map[string]string) (RWLayer, error)
+	CreateRWLayer(id string, parent ChainID, mountLabel string, initFunc MountInit, storageOpt map[string]string, sharedRootFs bool) (RWLayer, error)
```

Comment:
@dmcgowan I can try doing this. Just that I can't figure out what's the advantage of doing this. Currently `layerStore` is one which is implementing `Store` interface. And now it will also satisfy `ShareableStore` interface. So this will be useful only if some store was there which did not want to satisfy `ShareableStore` interface. And I am wondering where is that store which will satisfy `Store` but not `ShareableStore`.